tests/test_steering_vector.py

```diff
@@ -113,6 +116,57 @@ def test_SteeringVector_patch_activations_with_min_token_index(
     assert torch.equal(expected_hidden_state, patched_hidden_states[2][0, 5:])
 
 
+# verify that patch_activations works both when target indices is a list of indices or a mask
+@pytest.mark.parametrize(
+    "target_token_indices, non_target_token_indices",
+    [
+        ([2, 4, 7], [i for i in range(13) if i not in [2, 4, 7]]),
+        (
+            torch.tensor([1 if i in [2, 4, 7] else 0 for i in range(13)]),
+            [i for i in range(13) if i not in [2, 4, 7]],
+        ),
+    ],
+)
+@torch.no_grad()
+def test_SteeringVector_patch_activations_with_token_indices(
```

Comment:
👍  Great test! It would also be good to test the other types of inputs this can take (slice and Tensor). You can just copy/paste this test and change `target_token_indices` IMO

Reply:
I'm using pytest parametrization to do it all in one test now. Done in [9eb39c5](https://github.com/steering-vectors/steering-vectors/pull/21/commits/9eb39c5ed9b26e48d5e7bc05bf092befad50593c)

Reply:
nice! even cleaner